source/standards/breaking-down-work.html.md.erb

```diff
@@ -0,0 +1,36 @@
+---
+title: Breaking down work
+last_reviewed_on: 2024-10-10
+review_in: 6 months
+---
+# <%= current_page.data.title %>
+
+Working sustainably in small batches and performing regular releases is considered [industry best practice](https://en.wikipedia.org/wiki/DevOps_Research_and_Assessment#DORA_Four_Key_Metrics).
+It reduces the risk of errors being introduced, makes delivery of value more predictable, and enables us to learn faster.
+
+Our perception of value is not just putting working software and new features into the hands of users. It’s also:
+
+- Learning about user needs.
+- Improving operational resilience.
+- Minimising [toil](https://sre.google/sre-book/eliminating-toil/).
+- Reducing cost of change.
+
+We should aim to break down our work into independently releaseable pieces that each deliver their own value.
+
+## Why break down work?
+
+There are many reasons why working in smaller batches provides an advantage in software development.
+
+- It articulates value. If we are able to clearly state where the value is in a piece of work then we are more able to break it down into valuable milestones.
+- It reduces waste. Code that is written but not integrated and being used is wasted effort, and we want to minimise that waste.
+- It promotes predictable delivery. We often need to know when a feature is likely to be complete and having clear regular milestones facilitates those conversations with evidence rather than estimates.
+- It enables fast feedback. We can learn a lot from regularly releasing our code to production environments even if the effects are not visible to users. We also get feedback on our code design because well-written code will be easier to iterate on.
+- It boosts team morale. The unit of delivery is the team, and delivering value regularly is much better than working on big-bang releases for months at a time.
```

Comment:
Pedantic suggestion you can ignore if you would like.

The "what is better" is implied by the previous sentence, however I'd be curious to see how it reads if we change it to this:

"The unit of delivery is the team, and delivering value regularly is much better **for morale** than working on big-bang releases for months at a time"